links.ml

```diff
@@ -532,6 +538,14 @@ let whole_program_caching_main () =
   in
    Webif.serve_request envs prelude filename
 
+let construct_plugin_name x = 
+  "/home/harvey/links/plugins/ocamllib/ocaml_" ^ x ^ ".cmxs"
```

Comment:
Here is another absolute path dependency that needs to be removed.  Maybe use LIB_DIR as the starting point?